packages/docs/getting-started/running-a-validator.md

```diff
@@ -117,177 +133,170 @@ In all the commands we are going to see the `CELO_IMAGE` variable to refer to th
 
 ```bash
-docker stop celo-validator celo-proxy
+# On your local machine
+export CELO_VALIDATOR_SIGNER_POP=<YOUR-VALIDATOR-SIGNER-POP>
+export CELO_VALIDATOR_BLS_PUBLIC_KEY=<YOUR-VALIDATOR-BLS-PUBLIC-KEY>
+export CELO_VALIDATOR_BLS_SIGNATURE=<YOUR-VALIDATOR-BLS-SIGNATURE>
 ```
 
-And you can remove the containers (not the data dir) running:
+Let's stop the node from running for now:
 
 ```bash
-docker rm -f celo-validator celo-proxy
+# On the validator machine
+docker stop celo-validator
+docker rm celo-validator
 ```
 
-### Running the Docker containers in the background
+We'll get back to this machine later, but for now, let's give it a proxy.
 
-There are different option for executing Docker containers in the background. The most typical one is to use in your docker run commands the `-d` option. Also for long running processes, specially when you run in a remote computer, you can use a tool like [screen](https://ss64.com/osx/screen.html). It allows to connect and disconnect from running processes providing an easy way to manage long run processes.
+### Deploy a proxy
 
-It's out of the scope of this documentation to go through the `screen` options, but you can use the following command format with your `docker` commands:
+To avoid exposing the validator to the public internet, we are deploying a proxy node which is responsible to communicate with the network. On our Proxy machine, we'll setup the node as per usual now:
 
 ```bash
-screen -S <SESSION NAME> -d -m <YOUR COMMAND>
+# On the proxy machine
+mkdir -p celo-data-dir/proxy
+cd celo-data-dir/proxy
+docker run -v $PWD/proxy:/root/.celo $CELO_IMAGE init /celo/genesis.json
+docker run -v $PWD/proxy:/root/.celo --entrypoint cp $CELO_IMAGE /celo/static-nodes.json /root/.celo/
 ```
 
-For example:
+You can then run the proxy with
 
 ```bash
-screen -S celo-validator -d -m docker run --name celo-validator --restart always -p 127.0.0.1:8545:8545 .......
+# On the proxy machine
+docker run --name celo-proxy -d --restart always -p 30313:30303 -p 30313:30303/udp -p 30503:30503 -p 30503:30503/udp -v $PWD/proxy:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --proxy.proxy --proxy.proxiedvalidatoraddress $CELO_VALIDATOR_SIGNER_ADDRESS --proxy.internalendpoint :30503
 ```
 
-You can list your existing `screen` sessions:
+Once the proxy is running, we will need to retrieve it's enode so that the validator will be able to connect to it. 
 
 ```bash
-screen -ls
+# On the proxy machine, retrieve the proxy enode
+echo $(docker exec celo-proxy geth --exec "admin.nodeInfo['enode'].split('//')[1].split('@')[0]" attach | tr -d '"')
 ```
 
-And re-atach to any of the existing sessions:
+Now we need to set the proxy enode and proxy IP address in environment variables on the validator machine.
 
 ```bash
-screen -r -S celo-validator
+# On the validator machine
+export PROXY_ENODE=<proxy enode>
+export PROXY_IP=<proxy ip address>
```

Comment:
Where do we get this?

Reply:
Since the proxy and validator are separate machines, this should be just the IP address of the proxy machine itself.  @jcortejoso - can you please confirm that.